Documentation/install/all/ddns.md

```diff
@@ -0,0 +1,86 @@
+# Custom DNS service with RFC 2136 UPDATES ("DDNS")
+
+This document describes replacing the a cloud provider (Azure, AWS) DNS Terraform module with the DDNS module, allowing provisining configuration of custom, external DNS name servers supporting the [RFC 2136 `UPDATE` protocol][rfc2136].
```

Comment:
"...allowing configuration of external DNS servers, which support RFC 2136 -compliant updates"?